cpp/oneapi/dal/backend/common.hpp

```diff
@@ -94,6 +94,11 @@ inline constexpr Integer up_pow2(Integer x) {
     return power;
 }
 
+template <typename float_t>
+inline constexpr float_t exp_treshold() {
+    return std::is_same_v<float_t, double> ? -650.0 : -75.0f;
+}
```

Comment:
Recommend to create separate header for math functions, e.g., `dal/backend/math.hpp`

Reply:
Created a new math.hpp file and moved this function there